### ISSUE TITLE
[Meta] Project sustainability

### ISSUE THREAD

**Issue by brylie** [SustainableUrbanDesign/app#44]
We want to build this open-source project on a good foundation. To that end, we want to make sure there is clear documentation about how to get involved, community guidelines, funding, and more.

## Task
This is a meta issue, meaning other issues might stem from the basic goals of:
- support community building around the project
- help contributors, of all sorts, to get involved with the project
- obtain financial support for focused development

## Resources
- [Starting an Open Source Project](https://opensource.guide/starting-a-project/)
- [How to Run a Successful Free Software Project](https://producingoss.com/)
- [Make your community sustainable](https://opencollective.com/)

**Comment by brylie** [comment]
@MaxThakurCodes do you have any other resources or recommendations that are related to this issue?

**Comment by MaxThakurCodes** [comment]
Maybe getting a donation/funding system?

**Comment by brylie** [comment]
Ok, sounds good. Also, are you available for some pair programming this Saturday at 16:00 EET?

**Comment by MaxThakurCodes** [comment]
Yes, I should be able to. I just finished school for the school year, so I'll be able to refocus more time into this project!

**Comment by brylie** [comment]
@jfinmaniv mentioned [Agenda 21](https://en.wikipedia.org/wiki/Agenda_21). It sets some short-term sustainable development goals that might be good for us to follow.

**Comment by MaxThakurCodes** [comment]
@brylie Could you make an email for this project or would you like me to?
For Example, SustainableUrbanDevelopment@gmail.com.
